data/test/data_set.go

```diff
@@ -0,0 +1,14 @@
+package test
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/test"
+)
+
+func RandomSetID() string {
```

Comment:
is it worth `RandomDataSetID` ?? as the namespace is `test.RandomDataSetID` ... i was a little confused initially by the `Random` and `Set` ID ;)

Reply:
Ah, well, the namespace is really dataTest.RandomSetID, but I see your point. I'll be moving some of this stuff out anyways, so I'll leave it be for now.